README.md

```diff
@@ -1,2 +1,11 @@
 # ContributingToMyProject
+
 Writeup from maintainers, admins, contributors on how someone can get started with their project.
+
+Please follow the [contributing guide](CONTRIBUTING.md) to add your own contributing guide!
+
+| Project/Organization  | Author   |
+|:---------------------:|:---------:|
```

Comment:
The table will look much better if you align the text to the left side instead of center. Just replace `:---:` with `:---`

Everything looks good! Just some cosmetic suggestions. :smile: